Sources/App/Core/SearchLogger.swift

```diff
@@ -14,6 +14,18 @@
 
 import Foundation
 
+struct SearchFragment: Hashable, Codable {
+    let searchID: UUID
+    let query: String
+    let result: Search.Result?
```

Comment:
`Search.Result` is quite a complex object - which I don't think we should encode in full.

What data are we actually after here? It's package results only, isn't it? I think it would be best to start small and just send the package id or the canonical path for those and then extend with what's missing.

Reply:
Ah, I've only just now seen your full comment about stars etc.

It would be easy to pull those into a logging record type individually as needed. `Search.Result` can get quite big and there almost certainly is too much stuff in it.

I'll propose a change on top of your PR.

Reply:
BTW, do you have a link to your TestFlight app? I don't have it among my beta apps!

Reply:
SPISearch test flight link: https://testflight.apple.com/join/DaVCrzZp